python/SoapySDR.in.i

```diff
@@ -72,11 +73,28 @@ if os.name == 'nt' and hasattr(os, 'add_dll_directory'):
 ////////////////////////////////////////////////////////////////////////
 %include <SoapySDR/Config.h>
 
+////////////////////////////////////////////////////////////////////////
+// Check ABI before attempting to use Python module
+////////////////////////////////////////////////////////////////////////
+%include <std_string.i>
+
+%{
+#include <SoapySDR/Version.hpp>
+%}
+%include <SoapySDR/Version.hpp>
```

Comment:
You might be making swig process Version.hpp twice its also down later in the file. Not sure how they do include guards.

I think the `%insert("python")` or `%pythoncode %{` ends up at the bottom of the generated .py file anyone, so you dont need to change the other swig build stuff.

Reply:
I updated the PR to remove the duplicate Version.hpp checks later on. SWIG places the `%insert("python")` output in the output Python file relative to the generated code around it, so this intentionally places it before any attempt to reference the shared library.